### PR TITLE
Adds typings for keypair and licensekey

### PR DESCRIPTION
**Pull request by eskelter**
Adds typings for [keypair](https://www.npmjs.com/package/keypair)
Adds typings for [licensekey](https://www.npmjs.com/package/licensekey)

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/keypair/index.d.ts
Comment (plantain-00) [inline types/keypair/index.d.ts]: It's better to use its project url(https://github.com/juliangruber/keypair) rather than npm package url

**Review by plantain-00 — APPROVED** [review]

File: types/licensekey/index.d.ts
Comment (andy-ms) [inline types/licensekey/index.d.ts]: This documentation seems to just duplicate what the type already says.

File: types/licensekey/licensekey-tests.ts
Comment (andy-ms) [inline types/licensekey/licensekey-tests.ts]: Remove the commented-out code. And both `try-catch`es because we don't typecheck exceptions.

File: types/keypair/index.d.ts
Comment (andy-ms) [reply types/keypair/index.d.ts]: +1

**Comment by RyanCavanaugh** [comment]
Approved by third party. PR appears ready to merge pending review by a maintainer.

**Review by rbuckton — CHANGES_REQUESTED** [review]
@eskelter can you please address @andy-ms's comments?

**Comment by RyanCavanaugh** [comment]
@eskelter Please address comments from the code reviewers.

**Comment by DanielRosenwasser** [comment]
Hey @eskelter, the remaining work seems to be relatively small. If you can address the feedback, I can merge this in today.

**Comment by minestarks** [comment]
I'm closing this PR as it has pending revision requests and test failures which have been unaddressed in almost two months. If you'd still like to get these changes in, please open a new PR. Thanks!

**Pull request closed without merging**
